frontend/src/store/index.js

```diff
@@ -1010,6 +1020,198 @@ const getters = {
   },
   onlyShootsWithIssues (state, getters) {
     return getters['shoots/onlyShootsWithIssues']
+  },
+  projectNameByNamespace (state, getters) {
+    return ({ namespace } = {}) => {
+      const project = find(getters.projectList, ['metadata.namespace', namespace])
+      return get(project, 'metadata.name') || replace(namespace, /^garden-/, '')
```

Comment:
Do we still need this fallback?

Reply:
I don't know. But I don't wanted to start changing any code in this PR.

Reply:
ok